models/enums_and_bitflags.py

```diff
@@ -0,0 +1,237 @@
+"""
+Here we put all the dictionaries and convertion functions for getting/setting enums and bitflags.
+"""
+
+# This is the master format dictionary, that contains all file types for
+# all data models. Each model will get a subset of this dictionary.
+file_format_dict = {
+    1: 'fits',
+    2: 'hdf5',
+    3: 'csv',
+    4: 'json',
+    5: 'yaml',
+    6: 'xml',
+    7: 'pickle',
+    8: 'parquet',
+    9: 'npy',
+    10: 'npz',
+    11: 'avro',
+    12: 'netcdf',
+    13: 'jpg',
+    14: 'png',
+    15: 'pdf',
+}
+
+allowed_image_formats = ['fits', 'hdf5']
+image_format_dict = {k: v for k, v in file_format_dict.items() if v in allowed_image_formats}
+image_format_inverse = {v: k for k, v in image_format_dict.items()}
```

Comment:
Random musing : it's too bad python doesn't have a symmetric 1:1 mapping class.  As far as  I know, having an inverse dictionary is the only reasonable way to do it.

Reply:
We only need to calculate it once here, so it is not so bad.